Homework9/GenericClass.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Homework9
+{
+    public class GenericClass<T> where T : Human
+    {
+        public T[] array = { };
```

Comment:
Try to use more strict modifies if do not need an access to elements of class.
In this case you can use private  field - do not forget to follow naming rules